### DISCUSSION TITLE
Significantly lower in-hospital mortality in MIMIC-IV compared with MIMIC-III?

### DISCUSSION THREAD

**Discussion by cloverbunny** [MIT-LCP/mimic-code#1263] [MIMIC-IV] [answered]
I see that the in-hospital mortality from MIMIC-III is 11.5% (https://www.nature.com/articles/sdata201635), but when looking at the in-hospital mortality from MIMIC-IV, it seems to be lower. 
I'm looking at all of the `hospital_expire_flag` counts in the `admissions` table with `SELECT count(hospital_expire_flag) FROM `physionet-data.mimic_core.admissions` group by hospital_expire_flag` and get 0s: 514390 and 1s: 9350
Which leads to a hospital mortality rate of only 1.817%? 
Is there an explanation for this difference? I know there was a new rule that stopped allowing social security information for out of hospital mortality based on this thread (https://github.com/MIT-LCP/mimic-code/issues/1199) but wouldn't expect it to affect in-hospital mortality rates?
Am I misinterpreting the data? Is there somewhere else I should be pulling this information from?
Thanks!

PS - also noticed a discrepancy between `hospital_expire_flag` counts = 9350 and `discharge_location` = DIED counts = 9238 (small but not sure why they're not the same?)

**Comment by tompollard** [comment, marked as answer]
@cloverbunny the short answer is that the population has changed between MIMIC-III and MIMIC-IV. 

The subject_id list in the core MIMIC-IV `patients` table includes ED admissions. If you limit your analysis to ICU patients in MIMIC-IV, you should find the in-hospital mortality is comparable.

Another difference (though less relevant here) is out-of-hospital mortality, which is essentially not available for MIMIC-IV v1.0. The next release of MIMIC-IV should include out-of-hospital mortality obtained from state records.

**Reply by cloverbunny** [reply]
Awesome, thank you so much! 
`SELECT count(hospital_expire_flag) 
FROM `physionet-data.mimic_derived.icustay_detail` 
where first_icu_stay=true
group by hospital_expire_flag`
gives me 7360 deaths and 61851 alive = 10.6% mortality
Thanks especially for the super quick response!
